pennylane/labs/vibrational/christiansen_utils.py

```diff
@@ -848,7 +848,7 @@ def christiansen_integrals_dipole(pes, n_states=16):
     comm.Barrier()
     dipole_cform_onebody = comm.bcast(dipole_cform_onebody, root=0)
 
-    if pes.localized is True or pes.dipole_level > 1:
+    if pes.dipole_level > 1:
```

Comment:
It is fine for the PR context, but the function could be cleaned as it is simply a code block duplicated twice with these two cases.